pom.xml

```diff
@@ -1,76 +1,191 @@
-
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <modelVersion>4.0.0</modelVersion>
 
   <groupId>com.diffblue</groupId>
   <artifactId>deeptestutils</artifactId>
-  <version>1.0.0</version>
+  <version>0.0.1</version>
```

Comment:
Shouldn't this version be aligned with Diffblue Deeptest versions for ease of tracking?

Reply:
That's an interesting suggestion - however, there are a few arguments against this:

 - We want to release TG very frequently, we certainly don't want to release this as frequently (so the versions will drift and then go from 0.3.4 -> 0.9.10 which is a bit weird)
 - This is "released" so version 1.0.0 kind of makes sense

(Note the version will be changed to 1.0.0 when we release this branch).

Reply:
Ok. Makes sense.